man-db/PKGBUILD

```diff
@@ -3,6 +3,7 @@
 pkgname=man-db
 pkgver=2.7.6
 pkgrel=2
+epoch=1
```

Comment:
Git for Windows does not modify `man-db` Therefore this change should be unnecessary.